app/sql.js

```diff
@@ -1025,6 +1028,18 @@ async function removeAllFromTable(table) {
 
 // Conversations
 
+async function getSwarmNodesByPubkey(pubkey) {
+  const row = await db.get('SELECT * FROM conversations WHERE id = $pubkey;', {
+    $pubkey: pubkey,
+  });
+
+  if (!row) {
+    return null;
```

Comment:
should we return an empty array instead of null to keep the return type consistent?

Reply:
I was just mimicking what was done for the existing queries, not invested in it tho

Reply:
I'd rather keep it returning `null` and doing `|| []` where we call this instead of returning an empty array.